app/views/directives/bind-service/delete-binding-select-form.html

```diff
@@ -3,18 +3,18 @@ <h3 class="mar-top-none">
 </h3>
 <form name="ctrl.bindingSelection" class="mar-bottom-lg">
   <fieldset ng-disabled="ctrl.isDisabled">
-    <div ng-repeat="binding in ctrl.bindings" class="radio">
+    <div ng-repeat="binding in ctrl.bindings | orderBy: ctrl.firstAppForBindingName : binding" class="radio">
```

Comment:
Yeah that was definitely unneeded.  Didn't seem to do a flip, but the binding was passed as an arg anyway.  Removed in last push.